PF/PF-Assignment1/LinkedListStack.java

```diff
@@ -0,0 +1,72 @@
+package assignment1;
+
+public class LinkedListStack {
+	class Node{
+		int data;
+		Node next;
+		public Node(int item) {
+			data = item;
+			next = null;
+		}
+	}
+	
+	Node top= null;
+	
+	//Check whether stack is empty.
+	public boolean isEmpty() {
+		if(top==null) {
+			return true;
+		}
+		else
+			return false;
+	}	
+	
+	//Pushes item at the top stack
+	public void push(int item) {
+		if(isEmpty()==true) {
```

Comment:
This can be written as **if(isEmpty())**.

It is required to check empty condition, what if we don't check?